sycl-fusion/test/kernel-fusion/check-remapping.ll

```diff
@@ -91,6 +86,19 @@ declare !sycl.kernel.fused !13 !sycl.kernel.nd-ranges !15 !sycl.kernel.nd-range
 ; CHECK-NEXT:    ret void
 ; CHECK-NEXT:  }
 
+; CHECK-LABEL: define internal spir_func i64 @__global_linear_id_3_48_1_1_2_1_1(
+; CHECK-SAME:                                                                   ) #[[ATTRS]] {
+; CHECK-NEXT:    entry:
+; CHECK-NEXT:     %0 = call spir_func i64 @_Z33__spirv_BuiltInGlobalInvocationIdi(i32 2)
+; CHECK-NEXT:     %1 = mul i64 %0, 1
+; CHECK-NEXT:     %2 = call spir_func i64 @_Z33__spirv_BuiltInGlobalInvocationIdi(i32 1)
+; CHECK-NEXT:     %3 = mul i64 %2, 1
+; CHECK-NEXT:     %4 = add i64 %3, %1
+; CHECK-NEXT:     %5 = call spir_func i64 @_Z33__spirv_BuiltInGlobalInvocationIdi(i32 0)
+; CHECK-NEXT:     %6 = add i64 %5, %4
+; CHECK-NEXT:     ret i64 %6
+; CHECK-NEXT:   }
```

Comment:
Due to a bug when fusing more than one kernel in a module (not found when coming from the runtime), we are only using one fused kernel. Will fix in a different PR and update test adding the rest of the functions that should be generated.